squid/docker-compose.yml

```
@@ -1,15 +1,44 @@
 version: "3"
 
 services:
+  graphql-server:
+    container_name: graphql-server
+    build:
+      context: .
+      target: graphql-server
+    env_file: .env.example
```

This is still here, it shouldn't. if anything, then we should have the .env as environment file
```suggestion
    env_file: .env
```